engine/impl/Playlist.php

```diff
@@ -162,39 +164,86 @@ public function updatePlaylist($playlist, $date, $time, $description, $airname)
     }
 
     public function getTrack($id) {
-        $query = "SELECT tag, artist, track, album, label, id FROM tracks " .
+        $query = "SELECT created, tag, artist, track, album, label, id FROM tracks " .
                  "WHERE id = ?";
         $stmt = $this->prepare($query);
         $stmt->bindValue(1, (int)$id, \PDO::PARAM_INT);
         return $this->executeAndFetch($stmt);
     }
     
     public function getTracks($playlist, $desc = 0) {
-        $query = "SELECT tag, artist, track, album, label, id FROM tracks " .
+        $query = "SELECT created, tag, artist, track, album, label, id FROM tracks " .
                  "WHERE list = ? ORDER BY id";
         if($desc)
             $query .= " DESC";
         $stmt = $this->prepare($query);
         $stmt->bindValue(1, (int)$playlist, \PDO::PARAM_INT);
         return $this->execute($stmt);
     }
-    
-    public function insertTrack($playlist, $tag, $artist, $track, $album, $label) {
+
+    private static function logIt($msg) {
+        $out = fopen('php://stderr', 'w'); //output handler
+        fputs($out, $msg."\n"); //writing output operation
+        fclose($out);
+    }
+
```

Comment:
logIt needs to go.  Fine for debugging locally but not for production.  Remove related comment from README.md.  How developers manage their personal development is their own business, not part of the project.

Reply:
disagree, but removed from code. if you want people to work in the project it is good to give them a leg up on the nuts & bolts. i'll let you remove it from the readme if you feel strongly about it.

Reply:
Thank you for removing this.  There is a built-in PHP function error_log() that does exactly the same thing.  If you want to keep the comment in README.md, please revise it to refer to error_log.